toolkit/scripts/srpm_pack.mk

```diff
@@ -38,13 +38,11 @@ $(BUILD_SRPMS_DIR): $(STATUS_FLAGS_DIR)/build_srpms.flag
 	@echo Finished updating $@
 
 ifeq ($(DOWNLOAD_SRPMS),y)
-srpm_download_urls := $(SRPM_URL) $(SRPM_UPDATE_URL)
-
 $(STATUS_FLAGS_DIR)/build_srpms.flag: $(local_specs) $(local_spec_dirs) $(SPECS_DIR)
 	for spec in $(local_specs); do \
 		spec_file=$${spec} && \
 		srpm_file=$$(rpmspec -q $${spec_file} --srpm --define='with_check 1' --define='dist $(DIST_TAG)' --queryformat %{NAME}-%{VERSION}-%{RELEASE}.src.rpm) && \
-		for url in $(srpm_download_urls); do \
+		for url in $(SRPM_URL_LIST); do \
```

Comment:
Don't have a strong opinion here, but: Do we want to track the state of the URL list by adding a DEPENDS variable? ie it would trigger an SRPM re-download and re-build if the URL's changed? I'm not sure how much value there would be since the scheduler/optimizer will just ignore the new SRPMs anyways.

Reply:
I think for the reasons you mentioned there isn't much value add to tracking the state of the URL list, will leave as-is.